docs/egress.md

```diff
@@ -142,7 +142,6 @@ as the Egress IPs. A typical ExternalIPPool resource example:
     ipRanges:
     - start: 10.10.0.2
       end: 10.10.0.10
-    - cidr: 10.10.1.0/28
```

Comment:
Do we need to remove the CIDR here as it is valid?

Reply:
You are right. I just wanted to remove "-" before "apiVersion". Reverted the change here (but still remove "- cidr" from the the pool example in service-loadbalancer.md" to make it simpler).